corehq/apps/domain/tests/test_forms.py

```diff
@@ -191,3 +191,32 @@ def tearDown(self):
         OperatorCallLimitSettings.objects.all().delete()
         SMSAccountConfirmationSettings.objects.all().delete()
         super().tearDown()
+
+
+class TestAppReleaseModeSettingForm(TestCase):
+
+    def setUp(self) -> None:
+        domain = Domain.generate_name('test_domain')
+        self.domain_obj = Domain(name=domain)
+        self.domain_obj.save()
+
+    def test_release_mode_settings_visible_and_saved_without_error(self):
+        form = self.create_form(domain=self.domain_obj)
+        self.assertTrue('release_mode_visibility' in form.fields)
+
+        form.full_clean()
+        saved = form.save(Mock(), self.domain_obj)
+        self.assertEqual(True, saved)  # No error during form save
+
+    def create_form(self, domain=None, **kwargs):
```

Comment:
where there more tests planned or there were more tests earlier using this function?
I see only one use now, which passed the domain object and does not have any more kwargs, so looks like this could be simplified? You probably could remove this function now entirely

Reply:
Yes, tests that were not valid anymore were removed.
https://github.com/dimagi/commcare-hq/pull/32245/commits/ee34ec98d1d5784da48e31f98de77fc6d62547c7